webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/Login.scala

```diff
@@ -9,18 +9,26 @@ import scala.xml.NodeSeq
 class Login extends DispatchSnippet with DefaultExtendableSnippet[Login] {
 
   val userListProvider = RudderConfig.rudderUserListProvider
-
+//  Rudder server installation is complete! To get started, create a first user with:
   def mainDispatch = Map(
     "display" -> { authForm:NodeSeq =>
       if(userListProvider.authConfig.users.isEmpty) {
         <div class="col-xs-12" style="min-height:250px">
-          <div class="row" style="margin-top:50px">
-            <h3 class="text-danger text-center col-xs-10 col-xs-offset-1">No user is defined.</h3>
+          <div class="row" style="margin-top:30px">
+            <h3 class="text-center">
+              Rudder installation complete!
+              <i class="fa fa-check-circle" style="color: #9bc832;" aria-hidden="true"></i>
+            </h3>
+            <div class="text-center" style="margin-top:20px">
+                To get started, create a first user with:
```

Comment:
I would have put a blue `<i class="fa fa-info-circle" aria-hidden="true"></i>` on the left here